Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponAnimationController.cs

```diff
@@ -94,8 +92,11 @@ private void LateUpdate()
         {
             UpdateBobbingWeaponOnMovement();
             UpdateWeaponRecoil();
-
-            _weaponHolder.transform.localPosition = _weaponBobLocalPosition + _weaponRecoilLocalPosition;
+            
+            var camTransform = Camera.main.transform;
```

Comment:
Is it better if this is cached? even though it won't solve the issue.

Reply:
Yeah should definitely be cached, I'll fix that